proto/cosmos/store/commit_info.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package cosmos.store;
```

Comment:
I'd rather this not be a top-level `cosmos` package. Isn't this really something internal to the SDK that clients or even people trying to re-implement the SDK in say rust would never use? We should have something like `cosmos_sdk.internal` for stuff like this. Or am I wrong and is this an essential part of consensus state and the app hash?

Reply:
Note that after #6905, we'll have a `cosmos.base` parent, so this can go as `cosmos.base.store.v1beta1`

Reply:
Yes @aaronc, it's used to compute the `AppHash`. I don't really care where this goes. If you have a preference on location, just let me know and I'll move it.

Reply:
Let's put it where @amaurymartiny suggested

Reply:
ACK